build/installers/liferay-developer-studio/pom.xml

```diff
@@ -35,8 +35,8 @@
         <portal-version>7.0.10.2</portal-version>
         <portal-zip-name>liferay-dxp-digital-enterprise-tomcat-7.0-sp2-20170317165321327.zip</portal-zip-name>
         <plugin-sdk-download-base>https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public/com/liferay/portal/com.liferay.portal.plugins.sdk</plugin-sdk-download-base>
-        <plugin-sdk-zip-name>com.liferay.portal.plugins.sdk-1.0.9-withdependencies.zip</plugin-sdk-zip-name>
-        <plugin-sdk-version>1.0.9</plugin-sdk-version>
+        <plugin-sdk-zip-name>com.liferay.portal.plugins.sdk-1.0.11-withdependencies.zip</plugin-sdk-zip-name>
+        <plugin-sdk-version>1.0.11</plugin-sdk-version>
```

Comment:
Does the new version include some needed fixes?  Does it make it better for studio customers?